### DISCUSSION TITLE
Questions on usage

### DISCUSSION THREAD

**Discussion by saidhasyim** [bramses/bramses-highly-opinionated-vault-2023#10] [General]
Hi Bramses,

Thanks very much for this vault. I am using this for all my notes. I have read the readme and have a few questions.

1. 
`Finally highlight the text that you just created and cut/copy it to clipboard. Once in the clipboard hit ctrl-n to make a new note, and templater it will automatically create a Zettelkasten note. Make sure to go back to the note you created and link the newly processed note!`

At this stage after creating a Zettel, shall I delete the captured note from /Inbox/Computer Capture or /Smartphone Capture? Otherwise this list will pile up.

2. I currently store technical notes for each software (i.e. installation guide, command, etc). Where do you recommend me storing it? I saw that there are 2 possible places in BHOV: in Database/ or just create a new folder (i.e. /Software) at the root similar to /Readwise-Example. In what situation you would create a root folder vs create another folder in Databases?

3. In the case of technical notes, i am inclined not to link the text with anything else because i am afraid it will be difficult to search and mess up with the Graph View linking. Just like the example /CRM folder where Jane Doe and John Doe have no link inside (if that's intended). What do you think? 

4. I read in Readme the concept of keeping the unique id for each Zetel and also to keep the Zetel at the root folder. If the note is strictly related to a project, is it necessary to create a Zetel and link it to the Resources? Shouldn't we just create a new note under the /Resources folder, thereafter give it an easy-to-remember title? For example, in /Test Project 2/Resources, ther is a link to a Zetel at the root folder. Would you share with me what is the intention of creating a zetel for a note that is strictly for a specific project, rather than just create a non-zetel note and insert it inside Resources/? 

Thanks again for the wonderful writeup.

**Comment by bramses** [comment]
> At this stage after creating a Zettel, shall I delete the captured note from /Inbox/Computer Capture or /Smartphone Capture? Otherwise this list will pile up

I usually do, yes. The only exception is when the capture note time itself is important, like a food diary is a good example, knowing what you ate for dinner and at what time

> I currently store technical notes for each software (i.e. installation guide, command, etc). Where do you recommend me storing it? I saw that there are 2 possible places in BHOV: in Database/ or just create a new folder (i.e. /Software) at the root similar to /Readwise-Example. In what situation you would create a root folder vs create another folder in Databases?

My "rule" (there are no actual rules EOD do what you prefer 🙂) is that Databases is for structured files, things that have the same format. A good example is CRM or recipes (all recipes have ingredients and cook times for example). I could see Databases holding guides for sure, but if you prefer them as Resources it may make sense to put them in their own folder!

> In the case of technical notes, i am inclined not to link the text with anything else because i am afraid it will be difficult to search and mess up with the Graph View linking. Just like the example /CRM folder where Jane Doe and John Doe have no link inside (if that's intended). What do you think?

I think linking is fine. Again, to me, Database is just that, a rolodex of similar looking/feeling information. It is nice to have each be unique because of the Dataview on the homepage of the Databases, but link away if you feel the need to! However, if you find yourself referencing a certain page in your DB folder a lot, it may make more sense to "promote" it to a Zettel note.

> I read in Readme the concept of keeping the unique id for each Zetel and also to keep the Zetel at the root folder. If the note is strictly related to a project, is it necessary to create a Zetel and link it to the Resources? Shouldn't we just create a new note under the /Resources folder, thereafter give it an easy-to-remember title? For example, in /Test Project 2/Resources, ther is a link to a Zetel at the root folder. Would you share with me what is the intention of creating a zetel for a note that is strictly for a specific project, rather than just create a non-zetel note and insert it inside Resources/?

I've gone back and forth with this a lot too. The reason I usually prefer Zettel notes is that you can use the same "resource" in multiple projects, but I often find myself confused as to where I left things sometimes too! My Resources files are usually a collection of links and short blurbs from the Internet that are project related, Zettels are more reserved for Resources I add my own thoughts to and then I backlink them with the `projects` key to the Project in question. But, again, modify the system however you decide makes the most sense to your mind! BHOV is a jumping off framework for you to get the best work done you can, not a rigid locked in thing (though not to toot my own horn I would argue that its pretty optimized given the messiness of note taking and project management 😆)

**Comment by saidhasyim** [comment]
Hey Bram,

Thanks for all your clarifications. They make sense to me now. I have been using this template for >1 week now and am enjoying it.

I just have 1 more question that I'm unsure of its use. The "Thoughts I had today" in Daily. I have been thinking that this shall be a zettel note (captured through the "Capture Note" shortcut) rather than inside our daily task? Is this portion meant to capture the **when** this thought happened? If so, this means I need to build another "Computed" file which will query all "Thoughts I had today" in Daily, and also make sure i don't check the checkbox as it is not really a task. 

Let me know what you think?
